src/device.rs

```diff
@@ -1985,6 +1985,19 @@ impl DeviceRef {
         unsafe { msg_send![self, newTextureWithDescriptor: descriptor] }
     }
 
+    pub fn new_texture_with_descriptor_and_iosurface(
+        &self,
+        descriptor: &TextureDescriptorRef,
+        iosurface: io_surface::IOSurfaceRef,
+        plane: NSUInteger,
+    ) -> Texture {
+        unsafe {
+            msg_send![self, newTextureWithDescriptor: descriptor
+                                           iosurface: iosurface
+                                               plane: plane ]
```

Comment:
Looks like this file lacks any consistent formatting, but it should probably be:

```suggestion
            msg_send![self, newTextureWithDescriptor:descriptor
                                           iosurface:iosurface
                                               plane:plane]
```

Reply:
Resolved